cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/CaseExpressionsTest.scala

```diff
@@ -128,5 +131,23 @@ class CaseExpressionsTest extends DocumentingTest {
         resultTable()
       }
     }
+    section("Using the result of `CASE` in the succeeding clause or statement", "syntax-use-case-result") {
+      p("""You can use the result of `CASE` to set a property(s) or a label(s) on a node or relationship.
```

Comment:
I would rather use the plural form all the way:
```suggestion
      p("""You can use the result of `CASE` to set properties on a node or relationship.
```
and I find it particular that you pick the labels: We cannot bind a label to a variable as far as I know and if we bind the node/relationship to a variable, then we could do practically anything with that and not just assign a label.

Do you have a good example for that, that you could add?

Reply:
I don't have. I'll remove the label's part.